packages/server-wallet/src/protocols/__test__/open-channel.test.ts

```diff
@@ -51,10 +53,22 @@ test.each`
   ${prefundState1}    | ${0}    | ${prefundState0}    | ${prefundState1}    | ${funded}    | ${signState(postfundState2)}   | ${'when the prefund state is supported, and the channel is funded'}      | ${'signs the postfund state'}
   ${prefundState1}    | ${1}    | ${prefundState1}    | ${prefundState1}    | ${funded}    | ${signState(postfundState3)}   | ${'when the prefund state is supported, and the channel is funded'}      | ${'signs the postfund state'}
   ${prefundState0}    | ${0}    | ${prefundState0}    | ${prefundState0}    | ${notFunded} | ${fundChannelAction1}          | ${'when the prefund state is supported, and alice is first to deposit'}  | ${'submits transaction'}
-  ${altPrefundState0} | ${0}    | ${altPrefundState0} | ${altPrefundState0} | ${funded}    | ${fundChannelAction2}          | ${'when the prefund state is supported, and alice is secont to deposit'} | ${'submits transaction'}
+  ${altPrefundState0} | ${0}    | ${altPrefundState0} | ${altPrefundState0} | ${funded}    | ${fundChannelAction2}          | ${'when the prefund state is supported, and alice is second to deposit'} | ${'submits transaction'}
 `('$result $cond', ({supported, latest, latestSignedByMe, funding, action, myIndex}) => {
   const ps = applicationProtocolState({
-    app: {supported, latest, latestSignedByMe, funding, myIndex},
+    app: {
+      supported,
+      latest,
+      latestSignedByMe,
+      funding,
+      myIndex,
+      directFundingStatus: Channel.directFundingStatus(
+        supported,
+        funding,
+        participants[myIndex],
+        'Direct'
+      ),
+    },
```

Comment:
nit: If you defined
```
const directFundingStatus = Channel.directFundingStatus(
        supported,
        funding,
        participants[myIndex],
        'Direct'
      )
```
then this probably fits on one line.

```suggestion
    app: { supported, latest, latestSignedByMe, funding, myIndex, directFundingStatus },
```

Reply:
Unfortunately, it does not fit in one line 1a338b5c1